scripts/automation/process_one.py

```
@@ -0,0 +1,257 @@
+    print(f"  行 {row.row_index} を読み込み")
+
+    # 2. ステージ 3 完了済か検証
+    ok, msg = validate_stage_complete(row, force=args.force)
+    if not ok:
+        print(f"ABORT: {msg}", file=sys.stderr)
+        return 3
+
+    # 3. Doc URL 存在チェック
+    doc_url = row.get("Google Doc URL").strip()
+    if not doc_url and not args.skip_drive:
+        print(f"ABORT: Google Doc URL が空です", file=sys.stderr)
```

Ruff (0.15.17)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)